internal/cmd/gocdk/biome_add.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019 The Go Cloud Development Kit Authors
```

Comment:
Nit: I would consider naming this file `biome.go`; it'll probably make sense to put all of the biome-related commands in the same file, no?

Reply:
Turns out there is already a `biome.go`, maybe add it there?

Reply:
I had recommended to @clausti to use biome_add.go to avoid the conflict. biome.go has a bunch of biome-related support functions that are independent of any one command, and stuffing all the biome-related _commands_ in there will likely get messy.